src/_css/index.css

```diff
@@ -3,8 +3,43 @@
   src: url("/fonts/Nexa Bold.otf") format("opentype");
 }
 
+.float-logo-map-cloud {
+  animation: harmonic 8s infinite;
+  animation-timing-function: linear;
+}
+.float-cloud-back {
+  animation: harmonic 3s infinite;
+  animation-timing-function: linear;
+}
+
+.float-cloud-front {
+  animation: harmonic 6s infinite;
+  animation-timing-function: linear;
+}
+
+.w-wax_seal_submit_animation {
+  transition: transform 0.5s;
+}
+
+.w-wax_seal_submit_animation:hover {
+  transform: translateX(-50%) scale(1.2);
+}
+
+@keyframes harmonic {
+  0%,
+  100% {
+    transform: translateX(0);
+  }
+  25% {
+    transform: translateX(-1%);
+  }
+  75% {
+    transform: translateX(1%);
+  }
+}
```

Comment:
In general, we should not be writing custom CSS here. All new CSS should be registered with Tailwind, which auto-generates a lot of helpful utilities/modifiers and helps us keep a consistent design framework.

The documentation for doing that can be found here: https://tailwindcss.com/docs/animation#customizing, or I can convert the CSS over once the animation is tweaked.

Reply:
My say if just merge it for now, since it works fine I just want our site to look slightly more lively cuz we are going to announce it tomorrow. It kinda looks dead - Feel free to change it to something better when you make live site. I showed to some of my friends and they really liked how there is animation on it

After seeing the difference between linear movement and sine wave movement, the linear one looks a lot better IMO. Tina is the judge on that though. I made a codepen comparing the difference- you can easily swap the keyframe animation for the sine wave one.

https://codepen.io/jayleenli/pen/ExyWNeq